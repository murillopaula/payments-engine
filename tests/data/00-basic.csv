type,client,tx,amount
deposit,1,1,100.0
deposit,2,2,50.0
withdrawal,1,3,25.0
withdrawal,2,4,60.0
