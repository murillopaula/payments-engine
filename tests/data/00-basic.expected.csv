client,available,held,total,locked
1,75.0000,0.0000,75.0000,false
2,50.0000,0.0000,50.0000,false
